OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/DoPrivilegedCGAnalysis.scala

```diff
@@ -160,137 +160,132 @@ class DoPrivilegedCGAnalysis private[cg] (
     def analyze(p: SomeProject): PropertyComputationResult = {
         var analyses: List[DoPrivilegedMethodAnalysis] = Nil
 
-        val accessControllerType = ClassType("java/security/AccessController")
-        val privilegedActionType = ClassType("java/security/PrivilegedAction")
-        val privilegedExceptionActionType = ClassType("java/security/PrivilegedExceptionAction")
-        val accessControlContextType = ClassType("java/security/AccessControlContext")
-        val permissionType = ClassType("java/security/Permission")
-        val permissionsArray = ArrayType(permissionType)
+        val permissionsArray = ArrayType(ClassType.Permission)
```

Comment:
Some of these seem very specific. At least consider prefixing the class name with the package?